dotnet/aspnetcore/kubernetes/ArmTemplates/aks-template.json

```diff
@@ -30,8 +30,15 @@
         },
         "kubernetesVersion": {
             "type": "string"
+        },
+        "omsLocation": {
+            "type": "string"
         }
     },
+    "variables": {
+        "omsWorkspaceName": "[concat(parameters('clusterName'), '-', guid(parameters('clusterName')))]",
```

Comment:
can it exceed the max limit

Reply:
Also can it contain chars not allowed by omsworkspace name

Reply:
It can exceed the max limit, I'll trim the cluster name. the characters follow the same regex as the cluster so, it won't contain any invalid characters.